docs/internals/optimizer.rst

```diff
@@ -26,6 +26,10 @@ One can use ``solc --ir-optimized --optimize`` to produce an
 optimized Yul IR for a Solidity source. Similarly, one can use ``solc --strict-assembly --optimize``
 for a stand-alone Yul mode.
 
+.. note::
+    The `PeepholeOptimizer <https://en.wikipedia.org/wiki/Peephole_optimization>`_ and the inliner are always
```

Comment:
Are we referring to optimizer steps or just using generic names? I'd make that consistent one way or another.
```suggestion
    The `peephole optimizer <https://en.wikipedia.org/wiki/Peephole_optimization>`_ and the inliner are always
```

Reply:
The other occurrences of the word are as you suggested, so I am changing it.